tests/functional/cylc-install/07-no-recursive-installations.t

```diff
@@ -31,49 +31,33 @@ __HEREDOC__
 run_ok "$TEST_NAME_BASE" cylc validate "$PWD"/flow.cylc
 
```

Comment:
In this file I have cut down on duplication - no need to test that install into `cylctb/child` fails if you're also testing that install into `cylctb/child/grandchild` fails.